pkg/daemon/update.go

```diff
@@ -38,8 +39,8 @@ const (
 	coreUserName = "core"
 	// SSH Keys for user "core" will only be written at /home/core/.ssh
 	coreUserSSHPath = "/home/core/.ssh/"
-	// fipsCommand is the command to use when enabling or disabling FIPS
-	fipsCommand = "/usr/libexec/rhcos-tools/coreos-fips"
+	// fipsFile is the file to check if FIPS is enabled
+	fipsFile = "/proc/sys/crypto/fips_enabled"
```

Comment:
@cgwalters didn't approve of this method (should use the `coreos-fips --status` or whatever command): https://github.com/openshift/machine-config-operator/pull/1013/files#r307428010

Reply:
Yeah we removed that binary unfortunately, which is what lead to this

Reply:
`fipscheck`

Reply:
See https://github.com/openshift/machine-config-operator/issues/1250#issuecomment-551189317. That said, checking for `/proc/sys/crypto/fips_enabled` is a good start.

We don't want to use `coreos-fips` anymore. RHCOS has embedded support for turning on FIPS now.

Reply:
I think using the proc path is fine above is fine. `fipscheck` is an alternate possibility. `coreos-fips` has been removed.